build/web_compose/nginx.conf

```diff
@@ -0,0 +1,22 @@
+events {}
+
+http {
+  server {
+    listen 8080;
+
+    location / {
+        proxy_pass http://0.0.0.0:7070;
+        proxy_read_timeout 3600;
+    }
+
+    location /dcapi/ {
```

Comment:
@keyurva @pradh Any suggestions on this path naming?

This allows the custom DC to expose mixer endpoint through "host://dcapi/" but I would like a better name.

Reply:
Then how about:

Website:  custom.dc.org/app/api or custom.dc.org/viz/api
Mixer:  custom.dc.org/data/api or custom.dc.org/core/api

Reply:
@dwnoble any thoughts too?